File: AOJ/ITP1_re/itp_1_5_c.go
package main

import "fmt"

func main() {
	var h, w int
	for {
		fmt.Scan(&h, &w)
		if h == 0 && w == 0 {
			break
		}
		for i := 1; i <= h; i++ {
			for j := 1; j <= w; j++ {
				remi := i % 2
				remj := j % 2
				switch {
				case (remi == 1 && remj == 1) || (remi == 0 && remj == 0):
					fmt.Print("#")
				case (remi == 1 && remj == 0) || (remi == 0 && remj == 1):
					fmt.Print(".")
				}
			}
			fmt.Println()
		}
		fmt.Println()
	}
}

File: AOJ/ITP1_re/itp_1_5_c.cpp
#include <bits/stdc++.h>

int main() {
  int h, w, remi, remj;
  while (true) {
    scanf("%d %d", &h, &w);
    if (h == 0 && w == 0) {
      break;
    }
    for (int i = 1; i <= h; i++) {
      remi = i % 2;
      for (int j = 1; j <= w; j++) {
        remj = j % 2;
        if ((remi == 0 && remj == 0) || (remi == 1 && remj == 1)) {
          printf("#");
        } else {
          printf(".");
        }
      }
      printf("\n");
    }
    printf("\n");
  }
}
